### PR TITLE
Header: focus/tabbing + border jumpy content fix

### PR DESCRIPTION
**Pull request by Johnny-Scania**
**Describe pull-request**  
Fixing focus(tabbing) and issue with borders and "jumpy" content

<br/>
<br/>

**Solving issue**  
Fixes: 
#[AB1083](https://dev.azure.com/scaniadigitaldesignsystem/SDDS/_workitems/edit/1083)
#[AB1086](https://dev.azure.com/scaniadigitaldesignsystem/SDDS/_workitems/edit/1086)

<br/>
<br/>

**How to test focus(tabbing) issue**  
1. Run Storybook
2. Select Header component
3. Try to tab through header menu - it should show an outline around entire elements now

<br/>
<br/>

**How to test "jumpy" content issue**
1. Hover over different header menu options to see if the move for 1px left/right

  <br/>
<br/>

**Additional context**  
Short discussion DEVs + Alex

- introducing new classes vs updating customer
- UX when we code
- CSS simplification: one class name for header item/first-last-only child

### CODE REVIEW

File: components/src/patterns/side-menu/side-menu.stories.js [hunk 1]
```diff
@@ -75,16 +75,16 @@ const Template = ({
...
       <div class="sdds-navbar-side-menu expanded">
         <ul class="sdds-navbar-menu-list">
-          <li class="sdds-navbar-menu-item ${active ? 'active' : ''}">
-              ${icons}
+          <li class="sdds-navbar-menu-item ${
+            active ? 'active' : ''
+          }">              
               <a class="sdds-navbar-menu-item-link" href="#"> 
-              Item 1 
+              ${icons} Item 1
```
Comment (gaonita) [inline components/src/patterns/side-menu/side-menu.stories.js:82]: Screenshot 2021-11-01 at 11 24 48

somehow this makes menu icons disappear on collapse mode 🤔

Comment (Johnny-Scania) [reply components/src/patterns/side-menu/side-menu.stories.js:82]: Still working on it, I am trying to find a way so icons are also part of the anchor tag, meaning - clicking on them will also be a link, currently, they are outside of it.

Comment (Johnny-Scania) [reply components/src/patterns/side-menu/side-menu.stories.js]: @gaonita  - I think now it should be OK, please check when you find the time.

Comment (gaonita) [reply components/src/patterns/side-menu/side-menu.stories.js]: nice! the icons are back! just the alignment looks strange now 😭 alignment

Comment (Johnny-Scania) [reply components/src/patterns/side-menu/side-menu.stories.js]: Thanks for taking a look, I send the fix just a moment ago.

Comment (gaonita) [reply components/src/patterns/side-menu/side-menu.stories.js]: great! checked it, looks fine now✨

File: components/src/patterns/header/header.stories.js [hunk 2]
```diff
@@ -93,10 +93,10 @@ const toolbarMenuMobileHTML = `
       <a class="sdds-navbar-menu-item-link" href="#">EN <span class="sdds-icon-arrow"></span></a>
       <ul class="sdds-navbar-menu__dropdown-menu">
       <li class="sdds-navbar-menu__dropdown-item sdds-navbar-menu-item-description">Select language</li>
-        <li class="sdds-navbar-menu__dropdown-item active"><a href="#">English</a></li>
-        <li class="sdds-navbar-menu__dropdown-item"><a href="#">Finnish</a></li>
-        <li class="sdds-navbar-menu__dropdown-item"><a href="#">France</a></li>
-        <li class="sdds-navbar-menu__dropdown-item"><a href="#">Swedish</a></li>
+        <li class="sdds-navbar-menu__dropdown-item active"><a class="sdds-navbar-menu__dropdown-item-link" href="#">English</a></li>
```
Comment (Johnny-Scania) [inline components/src/patterns/header/header.stories.js:96]: @alexlioliopoulos  - here you can see that we are adding a new class to the anchor tag ( <a... ) in order to solve an issue with correct tabbing. That also allows users to click anywhere on the dropdown items and get redirected to the desired page. 

Why I am sharing this with you:
1) We need to update the code section on our website with the latest changes
2) Add this info to the release note/post/article so existing users can update accordingly

I am also writing this for myself not to forget - let's sync tomorrow around it.

**Comment by Johnny-Scania** [comment]
Issues it fixes on Header:
https://user-images.githubusercontent.com/91877950/139864800-697df0e8-878b-4dc0-8be6-6ecb8e431536.mp4

Issues it fixes on Side-menu:
https://user-images.githubusercontent.com/91877950/139864826-e91ae49a-8e28-4f71-8ac1-57e9bb83a2db.mp4

**Review by mmexvr — APPROVED** [review]
Looks good! 

- Reminder outline needs to be fixed on all

**Comment by alexlioliopoulos** [comment]
Is this including the quick fix around the proper alignment of the header items?

**Pull request merged**